airflow/providers/common/sql/hooks/sql.py

```diff
@@ -109,6 +109,8 @@ class DbApiHook(BaseForDbApiHook):
     connector = None  # type: Optional[ConnectorProtocol]
     # Override with db-specific query to check connection
     _test_connection_sql = "select 1"
+    # Override with the db-specific value used for placeholders
+    placeholder: str = "%s"
```

Comment:
Is it reference to [mandatory DB-API 2 paramstyle property](https://peps.python.org/pep-0249/#paramstyle)? 


```python
Python 3.9.10 (main, Feb 25 2022, 16:54:01) 
Type 'copyright', 'credits' or 'license' for more information
IPython 8.4.0 -- An enhanced Interactive Python. Type '?' for help.
PyDev console: using IPython 8.4.0
Python 3.9.10 (main, Feb 25 2022, 16:54:01) 
[Clang 13.0.0 (clang-1300.0.29.30)] on darwin

import sqlite3, trino, prestodb, psycopg2
sqlite3.paramstyle
Out[3]: 'qmark'
trino.dbapi.paramstyle
Out[4]: 'qmark'
psycopg2.paramstyle
Out[5]: 'pyformat'
prestodb.dbapi.paramstyle
Traceback (most recent call last):
  File "/Users/taragolis/.pyenv/versions/airflow-dev-env-39/lib/python3.9/site-packages/IPython/core/interactiveshell.py", line 3398, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-6-cb48c8aec499>", line 1, in <cell line: 1>
    prestodb.dbapi.paramstyle
AttributeError: module 'prestodb.dbapi' has no attribute 'paramstyle'
```